src/style.css

```diff
@@ -0,0 +1,113 @@
+
+:root {
+  --color-sh: #3d4e61;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir" ;
+  font-weight: 300;
+  font-style: normal;
+  src: local("Avenir"), url(/fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "AvenirBook" ;
+  font-weight: 300;
+  font-style: normal;
+  src: local("Avenir-Book"), url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "AvenirHeavy" sans-serif;
+  font-weight: 900;
+  font-style: normal;
+  src: local("Avenir-Heavy"), url(/fonts\Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 auto;
+  padding: 0 9px;
+  border: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.search_form {
+  width: 100%;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px #3d4e611a;
+}
+
+.form_input {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  color: var(--color-sh);
+}
+
+input {
```

Comment:
do not add such styles to the tag selector (use the tag selector only to reset some default styles, etc.)
so add class to each `input` and use it here